abci/example/js/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
what's this?

Reply:
this should be rebased on master or master merged into this branch, it seems you are ressurecting the abci server in python 👀 .

Reply:
sorry, not sure how I made that mistake but fixed it now - should just be the files relating to the change